src/features/contacts/index.tsx

```diff
@@ -51,7 +51,7 @@ export const Contacts = () => {
             <Avatar src={row.avatarUrl!}></Avatar>
           </ListItemAvatar>
           <ContactNameListItemText
-            primary={`${row.firstName} ${row.lastName}`}
+            primary={`${row.firstName ? row.firstName : ""} ${row.lastName ? row.lastName : ""}`}
```

Comment:
you can make it easier like
row.firstName || ""

Reply:
Fixed it.